lib/ansible/modules/system/ufw.py

```diff
@@ -197,6 +224,26 @@
     src: 2001:db8::/32
     port: 25
 
+- name: Deny all IPv6 traffic to tcp port 20 on this host
+  # this should be the first IPv6 rule
+  ufw:
+    rule: deny
+    proto: tcp
+    port: 20
+    to_ip: "::"
+    insert: 0
+    insert_relative_to: first-ipv6
+
+- name: Deny all IPv4 traffic to tcp port 20 on this host
+  # This should be the third to last IPv4 rule
```

Comment:
How that supposed to be third there?

Reply:
I guess it should have been `insert: -3`... I'll change that.

Reply:
Ah. No. `-1` is correct: this will add the new rule at the position where currently the second last IPv4 rule is (`insert: 0` would be position of last IPv4 rule, `insert: -1` the position of the second to last IPv4 rule), and since ufw will insert it before these two rules then, it will end up as the third to last.

Reply:
I guess it needs more documentation, though ;-)

Reply:
Third to last seems right here. But it is just confusing :)

Reply:
It is indeed! I hope with the new comment (see 689ebf0a7b31083b17799c8452b25023260572da) it is clearer. I guess most people would simply need "insert as first IPv6 command" (that's what I need this for), for example to insert an ALLOW from-ip before some DENYs, and there it's much less confusing...